integration/spark/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonLoadDataCommand.scala

```diff
@@ -108,7 +108,7 @@ case class CarbonLoadDataCommand(databaseNameOp: Option[String],
     // Delete stale segment folders that are not in table status but are physically present in
     // the Fact folder
     LOGGER.info(s"Deleting stale folders if present for table $dbName.$tableName")
-    TableProcessingOperations.deletePartialLoadDataIfExist(table, false)
+    // TableProcessingOperations.deletePartialLoadDataIfExist(table, false)
```

Comment:
same, if avoiding clean in load, please remove instead of commenting

Reply:
done